plugins/woocommerce/src/Internal/Admin/WcPaySubscriptionsPage.php

```diff
@@ -74,6 +71,56 @@ public function register_subscriptions_page() {
 		}
 	}
 
+	/**
+	 * Returns true if the store is eligible for the WooCommerce subsciptions empty state experiment.
+	 *
+	 * @return bool
+	 */
+	private function is_store_experiment_eligible() {
+		// Ineligible if WooCommerce Payments OR an existing subscriptions plugin is active.
+		$active_plugins         = PluginsHelper::get_active_plugin_slugs();
+		$plugin_ineligible_list = array(
+			'woocommerce-payments',
+			'woocommerce-subscriptions',
+			'subscriptions-for-woocommerce',
+			'subscriptions-for-woocommerce-pro',
+			'yith-woocommerce-subscription',
+		);
```

Comment:
Checking for existing plugins by slug. This list needs to be expanded to include additional paid plugins.

Reply:
@aprea @james-allan 
What are your thoughts on changing this to check for any plugins with `subscription` in the slug?

This may help cover plugins that we don't have access to yet (paid plugins) or that we haven't discovered.

For example:
**subscription**s-for-woocommerce
woocommerce-**subscription**s
yith-woocommerce-**subscription**

Reply:
> What are your thoughts on changing this to check for any plugins with subscription in the slug?

It's a neat idea and would solve the problems of maintaining this list, however, IMO it comes with a little risk given a subscription can refer to other things like a newsletter subscription, memberships and subscriptions can be used interchangeably sometimes so I'd be somewhat cautious about false positives. 

It's really a question about weighing the risk so we could ask this internally if there's a risk of serving this offer up to sites that might not strictly fit the criteria vs potentially false positives.

Reply:
I'll mark this as resolved since we now have included a slug for each of the 8 conflicting plugins dbe0f5f